admin-manual.md

```diff
@@ -26,18 +26,13 @@ Development Machine(s):
 - Eclipse
 - A merge tool such as Meld
 
-# Creating a Fork and Setting Configuration
+# Creating a Fork
 Use the GitHub interface or a private instance to create a forked repository.
 
-Proceed to set up a build environment, then edit
-`miso-web/src/main/webapp/WEB-INF/miso-config.xml` and set the naming schemes.
-The naming schemes will determine how MISO checks if object names (especially
```

Comment:
Naming scheme beans in `miso-web/src/main/webapp/WEB-INF/miso-config.xml` are interpolations of the `miso.X.naming.scheme` values set in the miso.properties file. That's why I've moved this part to line 122 below.